assets/js/app/example-app/components/ColorDisplay/ColorDisplay.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+const ColorDisplay = (props) => {
+  const { color } = props;
+
+  const style = {
+    color,
+    fontWeight: 'bold',
+    margin: '0px 5px 0px 5px',
```

Comment:
Is this duplicated in `ColorChanger/style.scss`?

Reply:
Yes, it is. I was having trouble making Jest run with external css/sass, it wasn't properly understanding it. Since I had been in it for too long, I've decided to put it in the file (although it's not the best practice, I know) for the moment, until I could find someone to help me with it :p